phoebe/atmospheres/passbands.py

```diff
@@ -375,7 +369,7 @@ def save(self, archive, overwrite=True, update_timestamp=True, export_inorm_tabl
         header['TIMESTMP'] = timestamp
         header['PBSET'] = self.pbset
         header['PBNAME'] = self.pbname
-        header['EFFWL'] = self.effwl
+        header['EFFWL'] = self.effwl.value
```

Comment:
is this already guaranteed to be in meters?  Or should we do `to_value` to double check (I see later that meters are assumed).

Reply:
as long as we aren't worried about ever deviating from that, then this is probably slightly cheaper.